src/services/helper/form.js

```diff
@@ -83,13 +86,20 @@ module.exports = class FormsHelper {
 
 	static async read(bodyData) {
 		try {
-			const form = await formsData.findOneForm(
-				bodyData.type,
-				bodyData.subType,
-				bodyData.action,
-				bodyData.ver,
-				bodyData.templateName
-			)
+			const key = bodyData.type + bodyData.subType + bodyData.action + bodyData.ver + bodyData.templateName
```

Comment:
@ankitpws getting key should be from function. we should not construct in all the places. and set form type to unique in model.